amr-wind/wind_energy/actuator/actuator_utils.cpp

```diff
@@ -44,6 +44,8 @@ std::set<int> determine_influenced_procs(
     std::set<int> procs;
     const int finest_level = mesh.finestLevel();
     const int nlevels = mesh.finestLevel() + 1;
+    // TODO bx should be mapped to unstretched coordinates here when using mesh
```

Comment:
Probably best to do the implementation inside `realbox_to_box` as I look closer.